src/PlusDataCollection/USDigitalEncodersTracking/vtkPlusUSDigitalEncodersTracker.h

```diff
@@ -146,11 +146,11 @@ class vtkPlusDataCollectionExport vtkPlusUSDigitalEncodersTracker : public vtkPl
 
 
 private:  // Variables.
-    bool coreXY;
+    bool coreXY = false;

```

Comment:
If you're making minor fixes to the class, would you consider moving all variables initialized in the header to the constructor's constructor list?

No biggie if you choose not to, but it would be appreciated.